examples/example_plotshape.m

```diff
@@ -1,10 +1,44 @@
+% Initialization.
 clear;
 setup;
 n = neuron.Neuron();
 n.reset_sections();
 
-axon = n.Section("axon");
-sl = n.SectionList();
-sl.append(axon);
-sl.printnames(); % outputs "axon"
-ps = n.PlotShape(sl);
+% Define topology.
+main = n.Section("main");
+branch1 = n.Section("branch1");
+branch2 = n.Section("branch2");
+branch3 = n.Section("branch3");
+main.set_diameter(4);
+branch1.set_diameter(3);
+branch2.set_diameter(2);
+branch3.set_diameter(1);
+main.length = 30;
+branch1.length = 10;
+branch2.length = 20;
+branch3.length = 5;
+branch1.connect(0, main, 1);
+branch2.connect(0, main, 1);
+branch3.connect(0, main, 0.5);
+main.nseg = 5;
+main(0.5).diam = 8;
+branch3.nseg = 2;
+branch3(1).diam = 3;
+n.topology();
+n.define_shape();
+
+% Make PlotShape of all Sections.
+ps_all = n.PlotShape(0);
```

Comment:
Question: right now, would this work with `n.PlotShape(false)`? If so, I'd prefer that to indicate there's something we're not doing (namely using interviews), rather than some magic number.

I guess the underlying issue is if logicals are translated to their underlying 0 or 1 value or not. (For Python, that's what we do... `False` and `True` get passed to NEURON as `0` and `1` respectively... and methods that generate booleans start out as 0 or 1, but these methods are known internally and so NEURON's 0s and 1s get converted to Falses and Trues before being received by Python.)

Reply:
I implemented automatic conversion from boolean to double if pushing to the NEURON stack.

Reply:
Nice. In that case, my suggestion is to change the zero here to false.